.env

```diff
@@ -1,4 +1,5 @@
-FAASM_VERSION=0.5.15
-CPP_CLI_IMAGE=faasm/cpp-sysroot:0.0.24
+FAASM_VERSION=0.5.16
+FAASM_CLI_IMAGE=faasm/cli:0.5.16
+CPP_CLI_IMAGE=faasm/cpp-sysroot:0.0.23
```

Comment:
Why has this version gone down? `0.0.24` is the latest isn't it?

Reply:
My mistake, thanks for spotting this.